bin/list-all

```diff
@@ -4,7 +4,7 @@ source "$(dirname "$0")/utils.sh"
 
 list_all() {
   install_or_update_perl_build
-  $(perl_build_path) --definitions | tr '\n' ' '
+  $(perl_build_path) --definitions | tr '\n' ' ' | tail -r
```

Comment:
`tail -r` isn't portable (only in BSD tail), and this pipe needs to come before the newlines are translated, I believe?  

For example, if `tac` is available, this does what's intended:

```
  $(perl_build_path) --definitions | tac | tr '\n' ' '
```

Reply:
Yes, that's a good point. `tac` would be preferable.